src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -0,0 +1,42 @@
+# Component Status
+
+Check back here anytime to see current component status information.
+If you have a suggestion for a new component not listed here, [create an issue on GitHub](https://github.com/mineral-ui/mineral-ui/issues) to let us know!
+
+See more detailed progress on our [Waffle.io board](https://waffle.io/mineral-ui/mineral-ui).
+
+<Legend />
+
+<!--
+
+Labels:
+~ new
+~ experimental
+
+Statuses:
+[available]
+[planned]
+[in development]
+[deprecated]
+
+-->
+
+| Component | Status |
+|---|---|
+| [Button](/components/button) | [available] |
+| [Card](/components/card) | [available] |
+| DateInput | [planned] |
+| [Dropdown](/components/dropdown) ~ new | [available] |
+| Flyout Navigation | [planned] |
```

Comment:
Maybe this should just be “Navigation”?

Reply:
yeah, +1 for keeping it vague until we know exactly what we're building.